.eslintrc.js

```diff
@@ -167,11 +167,12 @@ module.exports = {
     'no-restricted-globals': 0,
     'no-shadow-restricted-names': 1,
     'no-shadow': 1,
-    'no-undef-init': 0,
-    'no-undef': 1,
+    // these rules are disabled because the typescript compiler
+    // won't even build with undefined variables
+    'no-undef': 0,
```

Comment:
this rule should be disabled **only** for typescript files, I already disabled this one in my PR where I extended `eslint:recommended` config https://github.com/graphql/graphiql/pull/2507